src/main/webapp/default-site/scripts/rest/api/1/site/create.post.groovy

```diff
@@ -199,7 +205,7 @@ try {
         def context = SiteServices.createContext(applicationContext, request)
         try {
             if (!useRemote) {
-                SiteServices.createSiteFromBlueprint(context, blueprint, siteId, siteId, sandboxBranch, description,
+                SiteServices.createSiteFromBlueprint(context, blueprint, name, siteId, sandboxBranch, description,
```

Comment:
Should be siteId then siteName, throughout the system in terms of parameter order.